In [1]:
import pandas as pd

In [2]:
student_data={
    "Name":["Imran", "Kato", "Anderw", "Wassawa", "John"],
    "Maths":[89, 98, 84, 67, 56],
    "Science":[89, 98, 84, 67, 56],
    "gender":["male", "female", "male", "male", "female",],
}
students = pd.DataFrame(student_data)
students

,Name,Maths,Science,gender
0,Imran,89,89,male
1,Kato,98,98,female
2,Anderw,84,84,male
3,Wassawa,67,67,male
4,John,56,56,female


In [3]:
students["Maths"]

0    89
1    98
2    84
3    67
4    56
Name: Maths, dtype: int64

## ;MEAN
Finding the mean (findng the mean maths mark)

In [4]:
students["Maths"].mean()

np.float64(78.8)

In [5]:
students["Maths"].median()

np.float64(84.0)

## Find the mean Science mark and explain your results 
## Find the median of the Science mark and list the students attached to it 

In [6]:
students['Science'].mean()

np.float64(78.8)

after adding all the marks scored by all students and dividing by the tatol number of students i got my mean as (78.8)


In [13]:

students['Science'].std(),
students['Science'].var(),

(np.float64(289.70000000000005),)

## MEASURES diSIPATION
- STD
- VAR
- RANGE

## QN
- FIND THE SPREAD FOR SIC MARKS USING V
- WAT IS THE RANG OF THE SINCE MARKS AND COMENT ON U ANSWER

In [9]:
## variance of Science scores
students["Science"].var()

np.float64(289.70000000000005)